app/overrides/spree/admin/users/_form/add_enterprise_limit_form_element.html.haml.deface

```diff
@@ -1,5 +0,0 @@
-/ insert_bottom "div[data-hook='admin_user_form_fields'] div.alpha"
```

Comment:
Some screenshots in the PR description/comments would be nice to go with the various UI changes.

Reply:
Most of these changes don't change the UI at all. I was just importing from Spree and integrating our deface overrides into our view. But you are right, a screenshot would have been nice. I added one.